### PR TITLE
[lgwebos] Added rcButton channel

### PR DESCRIPTION
**Pull request by bbrodt**
This PR adds a new channel ("IRCode") to the LG WebOS binding, which simulates some of the button presses on the LG infrared remote control. Inspiration for this enhancement came [from the discussion here.](https://community.openhab.org/t/lgwebos-binding-sendbutton-question/85872/8)

Here is a list of IR codes that I have found to work. Note that this list has been compiled mostly through trial and error, and I may have missed some of the buttons. I have only been able to test this on an LGUJ657A TV so your mileage may vary.

Code String | Description
-- | --
LEFT | Left button in cursor control group
RIGHT | Right button in cursor control group
UP | Up button in cursor control group
DOWN | Down button in cursor control group
ENTER | "OK" button in the center of the cursor control group
BACK | "BACK" button
EXIT | "EXIT" button
0-9 | Number buttons
HOME | "HOME" button
RED | "RED" button
GREEN | "GREEN" button
YELLOW | "YELLOW" button
BLUE | "BLUE" button
PLAY | "PLAY" button
PAUSE | "PAUSE" button
STOP | "STOP" button

A sample HABPanel remote control widget can be found [in this github repository.](https://github.com/bbrodt/openhab2-misc)

### CODE REVIEW

**Comment by sprehn** [comment]
All lines in the files are marked as changed. I am unable to spot the differences. Did you change newlines, or sth?

File: bundles/org.openhab.binding.lgwebos/README.md [hunk 1]
```diff
@@ -62,6 +62,33 @@ Thing lgwebos:WebOSTV:tv1 [host="192.168.2.119", key="6ef1dff6c7c936c8dc5056fc85
 | mediaPlayer     | Player    | Media control player                                                                                                                                                                                                    | W          |
 | mediaStop       | Switch    | Media control stop                                                                                                                                                                                                      | W          |
 | appLauncher     | String    | Application ID of currently running application. This also allows to start applications on the TV by sending a specific Application ID to this channel.                                                                 | RW         |
+| rcButton        | String    | Simulates pressing of a button on the TV's infrared remote control. See below for a list of button names.                                                                                                               | W          |
```
Comment (sprehn) [inline bundles/org.openhab.binding.lgwebos/README.md]: still mentions "infrared". Would leave it out. Just mentione ... on the TV's remote control.

**Comment by Hilbrand** [comment]
The files all have windows line endings that causes the unwanted differences.

**Comment by bbrodt** [comment]
Dammit! Sorry about that :-( Should I change the line endings and try again?

**Comment by Hilbrand** [comment]
Yes please change the line endings.

**Comment by Hilbrand** [comment]
Only for the files you changed please....

**Comment by bbrodt** [comment]
Abandoned after EOL change screwup :-/

**Pull request closed without merging**